Solution-Accelerator/README.md

```diff
@@ -1,37 +1,91 @@
-## Testing NVIDIA-NIM Unit Tests with Solution Accelerator
+# NVIDIA-NIM Unit Tests with Solution Accelerator
 
-**Prerequisites**
+This guide explains how to test NVIDIA-NIM unit tests using the Solution Accelerator.
 
-Repository Setup: Know the runtime image path and tag for solution accelarator application on the cluster.
+## Prerequisites
 
-**Steps** 
+- Access to the Solution Accelerator platform
+- Runtime image path and tag for the solution accelerator application
+- Cluster access with appropriate permissions
 
-**Import Tar File:**
+## Setup Process
 
-- Go to Get started to access Solution accelerator interface.
-- Click on Add New.
-- Upload the tar file containing the NVIDIA-NIM unit tests.
-- Fill in the solution accelerator details, if deploying in a non-user namespace add label **hpe-ezua/ezmodels=true** to the namespace
-- Configure ClusterServingRuntime Image:
+### 1. Import Tar File
+1. Navigate to the Tools & Framework section
+2. Click "Import Framework"
+3. Upload your NVIDIA-NIM unit tests tar file
```

Comment:
can we rename it , upload charts instead of NVIDIA-NIM

Also, can we have helm-charts source?